### ISSUE TITLE
MainView intercepting clicks

### ISSUE THREAD

**Issue by akhahaha** [umano/AndroidSlidingUpPanel#264]
I am attempting to implement the panel within a view that needs to implement a custom onTouchListener. However, the panel main view seems to be intercepting clicks. Is there a way to disable this? I've tried playing around with setDragView(), but I'm not quite sure what it does and it doesn't seem to be working.

Of interesting note however, is that the panel works fine with the ViewPager, so perhaps it might be an issue with how the other listeners are implemented.

Thank you for providing such a great feature to the community!

**Comment by squinney** [comment]
Have been using this fantastic library for a recent project and this thread helped me to correct a bug where clicks wouldn't pass through to the parent when the SlidingUpPanelLayout was inside a child of a ViewAnimator by adding the change from akhahaha's post.  After recently moving to a gradle build, I ran into a bunch of trouble getting the library project to work within my main project (largely due to my unfamiliarity with anything gradle).  Ultimately I was able to get the library project to work through modifications of the gradle script files but was wondering/hoping if this change will be merged in to a release such that I can eliminate my modified library project altogether and just reference the gradle dependency?

Thanks!
